webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -92,6 +92,11 @@ def json_dumps(data):
     return jinja2.Markup(json.dumps(data))
 
 
+@library.filter
+def to_json(data):
+    return json.dumps(data)
```

Comment:
I know we're not using this `to_json` to in a inline script block. We're using inside a data tag. However, there might be other vulnerabilities to do with data tags. Or perhaps we forget and use it in some dangerous way in some other jinja template.